src/catalog/tests/open.rs

```diff
@@ -183,27 +184,27 @@ async fn get_deployment_generation<D: DurableCatalogState>(
 
 #[mz_ore::test(tokio::test)]
 #[cfg_attr(miri, ignore)] //  unsupported operation: can't call foreign function `TLS_client_method` on OS `linux`
-async fn test_open_check() {
+async fn test_open_savepoint() {
     let (debug_factory, stash_config) = stash_config().await;
     let openable_state = stash_backed_catalog_state(stash_config);
-    open_check(openable_state).await;
+    open_savepoint(openable_state).await;
     debug_factory.drop().await;
 }
 
 #[mz_ore::test(tokio::test)]
 #[cfg_attr(miri, ignore)] //  unsupported operation: can't call foreign function `TLS_client_method` on OS `linux`
-async fn test_debug_open_check() {
+async fn test_debug_open_savepoint() {
     let debug_factory = DebugStashFactory::new().await;
     let debug_openable_state = debug_stash_backed_catalog_state(&debug_factory);
-    open_check(debug_openable_state).await;
+    open_savepoint(debug_openable_state).await;
     debug_factory.drop().await;
 }
 
-async fn open_check<D: DurableCatalogState>(
+async fn open_savepoint<D: DurableCatalogState>(
```

Comment:
This is being renamed because...it was already using a savepoint and we forgot to update the fn name?

Reply:
Yes, this is the test function that tests the `open_savepoint` method. Originally I called the method `open_check`, but I renamed it and forgot to rename the tests.